docs/contributing-to-airbyte/resources/developing-locally.md

```diff
@@ -1,282 +0,0 @@
-2. Clone the fork on your workstation:
-
-If developing connectors, you can work on connectors locally but additionally start the platform independently locally using :
-
-```bash
-git clone git@github.com:{YOUR_USERNAME}/airbyte.git
-cd airbyte
-./run-ab-platform.sh
-```
-
-If developing platform:
-
-```bash
-git clone git@github.com:{YOUR_USERNAME}/airbyte-platform.git
-cd airbyte-platform
-docker compose up
-```
-
-## Build with `gradle`
-
-To compile and build the platform, run the following command in your local `airbyte-platform` repository:
-
-```bash
-./gradlew build
-```
-
-This will build all the code and run all the unit tests.
-
-`./gradlew build` creates all the necessary artifacts \(Webapp, Jars and Docker images\) so that you can run Airbyte locally. Since this builds everything, it can take some time.
-
-:::info
-
-Optionally, you may pass a `VERSION` environment variable to the gradle build command. If present, gradle will use this value as a tag for all created artifacts (both Jars and Docker images).
-
-If unset, gradle will default to using the current VERSION in `.env` for Jars, and `dev` as the Docker image tag.
-
-:::
-:::info
-
-If running tasks on a subproject, you must prepend `:oss` to the project in gradlew. For example, to build the `airbyte-cron` project the command would look like: `./gradlew :oss:airbyte-cron:build`.
-
-:::
-
-:::info
-
-Gradle will use all CPU cores by default. If Gradle uses too much/too little CPU, tuning the number of CPU cores it uses to better suit a dev's need can help.
-
-Adjust this by either, 1. Setting an env var: `export GRADLE_OPTS="-Dorg.gradle.workers.max=3"`. 2. Setting a cli option: `./gradlew build --max-workers 3` 3. Setting the `org.gradle.workers.max` property in the `gradle.properties` file.
-
-A good rule of thumb is to set this to \(\# of cores - 1\).
-
-:::
-
-:::info
-
-On Mac, if you run into an error while compiling openssl \(this happens when running pip install\), you may need to explicitly add these flags to your bash profile so that the C compiler can find the appropriate libraries.
-
-```bash
-export LDFLAGS="-L/usr/local/opt/openssl/lib"
-export CPPFLAGS="-I/usr/local/opt/openssl/include"
-```
-
-:::
-
-## Run in `dev` mode with `docker-compose`
-
-These instructions explain how to run a version of Airbyte Platform that you are developing on (e.g. has not been released yet).
-
-In your local `airbyte-platform` repository, run the following commands:
-
-```bash
-./gradlew build
-VERSION=dev docker compose up
-```
-
-The build will take a few minutes. Once it completes, Airbyte compiled at current git revision will be running in `dev` mode in your environment.
-
-If you are running just connectors, you don't need the first step:
-
-```bash
-VERSION=dev docker compose up
-```
-
-In `dev` mode, all data will be persisted in `/tmp/dev_root`.
-
-## Add a connector under development to Airbyte
```

Comment:
is there an abctl equivalent to this section?

Reply:
Not yet, it is on the abctl board though and we hope to get to it in the next ~week. Will update the documentation then.

Reply:
Sounds good. Mainly for the connector teams, so non-blocking for me.